jade/index.jade

```diff
@@ -0,0 +1,61 @@
+doctype html
+html(lang='en')
+  head
+    meta(charset='utf-8')
+    meta(name='description', content='meta desc')
+    meta(name='viewport', content='width=device-width')
+    title Avsc
+    link(rel='stylesheet', href='../stylesheets/main.css', type='text/css')
+  
+  body
+    script(src='../js/bundle.js')
```

Comment:
In case you were wondering if it matters where `script` tags go, [this question](http://stackoverflow.com/q/436411/1062617) is a good read.